controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const { readTopics } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+  readTopics()
+    .then((rows) => {
+      res.status(200).send(rows);
```

Comment:
Should be sent with descriptive names i.e. topics.  You can send as an object, {topics}. That way it will be on the response body under the descriptive key of topics.